profile.html

```diff
@@ -0,0 +1,284 @@
+<!Doctype html>
+<title> "My Portfolio" </Title>
+<head>
+<style>
+:root {
+  --main-white: #f0f0f0;
+  --main-red: #be3144;
+  --main-blue: #45567d;
+  --main-gray: #303841;
+  }
+  *{
+  margin: 0;
+  padding: 0;
+  box-sizing:border-box;  
+}
+*::before,
+*::after {
+  box-sizing: border-box;
+} /* i changed here*/
+
+html {
+ /* box-sizing: border-box; no need to write again*/
+
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+}
+@media (max-width: 75em) {
+  html {
+    font-size: 60%;
+  }
+}
+@media (max-width: 61.25em) {
+  html {
+    font-size: 58%;
+  }
+}
+@media (max-width: 28.75em) {
+  html {
+    font-size: 55%;
+  }
+}
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1.8rem; /* 18px */
+  font-weight: 400;
+  line-height: 1.4;
+  /*color: var(--main-white);*/
+  color:black;
+}
+
+h1,
+h2 {
+  font-family: 'Raleway', sans-serif;
+  font-weight: 700;
+  text-align: center;
+}
+
+h1 {
+  font-size: 6rem;
+}
+
+h2 {
+  font-size: 4.2rem;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: var(--main-white);
+}
+
+img {
+  /*display: block;
+  width: 100%;
+*/
+  max-width: 100%;}
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  position: sticky;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background: var(--main-blue);
+  box-shadow: 0 2px 0 rgba(0, 0, 0, 0.4);
+  z-index: 999;
+}
+.nav-list {
+  display: flex;
+  margin-right: 2rem;
+}
+
+@media (max-width: 28.75em) {
+  .nav {
+    justify-content: center;
+  }
+
+  .nav-list {
+    margin: 0 1rem;
+  }
+}
+
+.nav-list a {
+  display: block;
+  font-size: 2.2rem;
+  padding: 2rem;
+}
+
+.nav-list a:hover {
+  background: var(--main-blue);
+}
+
+.welcome-section{
+  height:100vh;
+    width:100%;
+  background-color:;/* u can add some color*/
+    padding:10rem 10rem 2rem 10rem;
+    margin-top: auto;
+
+}
+.welcome-section h1,p{
+  text-align: center;
+}
+/*here also missing ur section, u need to write here some stuff, also add some margins n paddings*/
+.projects-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  grid-gap: 2rem;
+  width: 100%;
+  max-width: 1280px;
+  margin: 0 auto;
+  margin-bottom: 6rem;
+}
+
+@media (max-width: 30.625em) {
+  .projects-section {
+    padding: 6rem 1rem;
+  }
+
+  .projects-grid {
+    grid-template-columns: 1fr;
+  }
+}
+
+.project {
+  background: var(--main-gray);
+  box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
+  border-radius: 2px;
+}
+
+.code {
+  color: var(--main-gray);
+  transition: color 0.3s ease-out;
+}
+
+.project:hover .code {
+  color: #ff7f50;
+}
+
+.project-image {
+  height: 50%;
+  width: 50%;
+  object-fit: cover;
+}/* here provide space for other projects */
+
+.project-title {
+  font-size: 2rem;
+  padding: 2rem 0.5rem;
+}
```

Comment:
Instead of using relative measurement, using pixels will be easy to understand for other people. 
![855](https://user-images.githubusercontent.com/100348154/178299368-0cdb790f-43b6-4561-8bda-34bb4130e769.gif)